_core/thread/post.php

```diff
@@ -41,6 +33,20 @@ function formatOP() {
         $com = $this->abbr($com, MAX_LINES_SHOWN, $no, $no); //lol
         $com = $this->auto_link($com, $no);
 
+        
+        $image = new Image;
+        $image->inIndex = $this->inIndex;
+
+        if ($media != null) {
+            $files = json_decode($media, true);
+            $xls = (count($files) > 1) ? "file multi-op" : "file no-multi";
```

Comment:
If we're rolling our own class names (since we probably aren't referencing) we should go with `op-multi` and `op-solo` (or equivalent) to make selecting both in CSS easier.